lib/fastlane/actions/oclint.rb

```diff
@@ -60,11 +85,22 @@ def self.available_options
                                        description: 'The json compilation database, use xctool reporter \'json-compilation-database\'',
                                        default_value: 'compile_commands.json',
                                        optional: true),
+          # Deprecated, remove as soon as possible
```

Comment:
Everything below or just this one option? If just this one option, this should be clearer